testing/example_scripts/fixtures/fill_fixtures/test_conftest_funcargs_only_available_in_subdir/sub2/conftest.py

```diff
@@ -6,4 +6,4 @@
 
 @pytest.fixture
 def arg2(request):
-    pytest.raises(Exception, request.getfixturevalue, "arg1")
+    pytest.raises(Exception, request.getfixturevalue, "arg1")  # noqa: B017
```

Comment:
For linters that don't have human-readable rule identifiers, it'd be a good idea to add a code comment explaining what the suppressions hide and when they can be dropped.